CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Drop official support for Go 1.10. ([#44](https://github.com/pellared/taskflow/pull/44))
+- Task parameters need to be registered, both with `Taskflow.Configure*()` and `Task.Parameters`.
```

Comment:
I wanted to comment about it in the previous PR but I did not have any good idea nor hint how to address my concern...

I do not like this `Configure` prefix. 


When I am calling `ConfigureString`, I think that I am not configuring any string. I am adding/registering a NEW  parameter/flag. Do you have any other proposals for names? Maybe simply `AddStringParam` instead?

My other idea is to make some kind of more complex structure which one could use like following:
```golang
param := flow.Params.AddString(...)
task := flow.Tasks.Add(...)
```
however, I feel this might be overengineered

What do you think?

Reply:
I'm with you, reviewing all documentation, it's always written as "registers X parameter" - so I've changed the name to `RegisterXParam`, which also matches the return type.

Reply:
Then I think you just need to update README and we can resolve this comment

Reply:
Ah, I think I missed that last comment - thank you for doing this.